frontend/packages/integration-tests-cypress/views/list-page.ts

```diff
@@ -0,0 +1,52 @@
+export const listPage = {
+  titleShouldHaveText: (title: string) =>
+    cy.byLegacyTestID('resource-title').should('have.text', title),
+  projectDropdownShouldExist: () => cy.byLegacyTestID('namespace-bar-dropdown').should('exist'),
+  projectDropdownShouldContain: (name: string) =>
+    cy.byLegacyTestID('namespace-bar-dropdown').contains(name),
+  projectDropdownShouldNotExist: () =>
+    cy.byLegacyTestID('namespace-bar-dropdown').should('not.exist'),
+  clickCreateYAMLButton: () => {
+    return cy
+      .contains('button', 'Create')
```

Comment:
This seems fragile. I'd feel much better about it if we used a `data-test` attribute. If there is any other button on the page with the text "Create" in it, this is no longer unique.

Reply:
fixed